psuutil/main.py

```diff
@@ -172,7 +171,8 @@ def status(index, textonly):
         msg = ""
         psu_name = "PSU {}".format(psu)
         if psu not in supported_psu:
-            status_table.append([psu_name, "NOT SUPPORTED"])
+            click.echo("Error! The {} is not available on the platform.\n" \
+            "Number of supported PSU - {}.".format(psu_name, platform_psuutil.get_num_psus()))
```

Comment:
Change to `"Number of supported PSU: {}"`

Replacing hyphen and preceding space with a colon and removing the trailing period will make parsing the output easier, if need be.

Reply:
Too late; already merged. Can modify later.

Reply:
Thanks.
It does not needed now.